Source/Codecov/Utilities/EnviornmentVariable.cs

```diff
@@ -9,5 +10,20 @@ internal static string GetEnviornmentVariable(string name)
             var env = Environment.GetEnvironmentVariable(name);
             return string.IsNullOrWhiteSpace(env) ? string.Empty : env;
         }
+
+        internal static string GetFirstExistingEnvironmentVariable(params string[] names)
+        {
+            foreach (var name in names)
+            {
+                var env = Environment.GetEnvironmentVariable(name);
+
+                if (env != null)
```

Comment:
should really use `string.IsNullOrWhiteSpace`.

I can be set, but without a value which would then be an empty string (or so I assume).